src/window.ts

```diff
@@ -0,0 +1,2 @@
+export type WindowOnlyOp = 'row_number' | 'rank' | 'dense_rank' | 'percent_rank' | 'cume_dist'
```

Comment:
Why not adding this to transform.ts?  

(It's kinda weird that you add this file only for this.)  

If you wanna have window.ts,  better have one file for each transform type. (I don't think we should though -- at least for now.)

Reply:
I actually didn't realize this was in a separate file. I moved it into transform.ts

Reply:
Actually, I had done this because WindowOnlyOp is necessary for vega.schema.ts. I think I would create a circular dependency if I had it in transform.ts, so I moved it to transform.d.ts